copier/copier.go

```diff
@@ -988,11 +988,16 @@ func copierHandlerStat(req request, pm *fileutils.PatternMatcher) *response {
 		s := StatsForGlob{
 			Glob: req.preservedGlobs[i],
 		}
-		stats = append(stats, &s)
 		// glob this pattern
 		globMatched, err := filepath.Glob(glob)
 		if err != nil {
+			if os.IsExist(err) && strings.ContainsAny(glob, "*?[") {
```

Comment:
The godoc for `filepath.Glob()` says that the only error it returns is `filepath.ErrBadPattern`.  Does this test ever succeed?